### PR TITLE
Change bg to background in semantic token

### PR DESCRIPTION
**Pull request by jessicawoodin**
## Chromatic

https://238-vads-mobile-bg-token--65a6e2ed2314f7b8f98609d8.chromatic.com

---
## Configuring this pull request
- [x] Link to any related issues in the description so they can be cross-referenced.
- [x] Add the appropriate version patch label (`major`, `minor`, `patch`, or `ignore-for-release`).
    - See [How to choose a version number](https://github.com/department-of-veterans-affairs/component-library#how-to-choose-a-version-number) for guidance.
    - Use `ignore-for-release` if files haven't been changed in a component library package. (ie. only Storybook files)
- [ ] DST Only: Increment the `/packages/core` version number if this will be the last PR merged before a release.
- [x] Complete all sections below.
- [ ] Delete this section once complete

## Description
- Changed `vads-color-bg-default-on-dark` to `vads-color-background-default-on-dark`
- Closes https://github.com/department-of-veterans-affairs/va-mobile-library/issues/238

## QA Checklist
- [ ] Component maintains 1:1 parity with design mocks
- [ ] Text is consistent with what's been provided in the mocks
- [ ] Component behaves as expected across breakpoints
- [ ] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [ ] Tab order and focus state work as expected
- [ ] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [ ] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [ ] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Acceptance criteria
- [ ] QA checklist has been completed
- [ ] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by micahchiang — APPROVED** [review]

File: packages/css-library/dist/tokens/css/variables.css [hunk 1]
```diff
@@ -11,7 +11,7 @@
   --small-desktop-screen: 1008px;
   --large-screen: 1201px;
   --color-link-default-hover: rgba(#000000, 0.05);
-  --vads-button-color-background-primary-alt-active-on-light: #154c21;
+  --vads-button-color-background-primary-alt-active-on-light: #0a200f;
```
Comment (jamigibbs) [inline packages/css-library/dist/tokens/css/variables.css:14]: Is this hex color supposed to change?

Comment (micahchiang) [reply packages/css-library/dist/tokens/css/variables.css:14]: huh, that is weird. Where is that coming from :thinking:

Comment (micahchiang) [reply packages/css-library/dist/tokens/css/variables.css]: I rebuilt the tokens in dist and it reverted this :shrug:

Comment (jamigibbs) [reply packages/css-library/dist/tokens/css/variables.css]: That is so weird. 😆

**Pull request merged**